artemis/modules/nuclei.py

```diff
@@ -498,6 +581,44 @@ def run_multiple(self, tasks: List[Task]) -> None:
                 )
             )
 
+            dast_targets.clear()
+            for keyword, template_data in DAST_SCANNING.items():
+                dast_targets[keyword] = []
+                for item in link_package:
+                    if item:
+                        param_url = add_common_params_from_wordlist(
+                            item, template_data["params_wordlist"], template_data["param_default_value"]
+                        )
+                        dast_targets[keyword].append(param_url)
+
+                findings.extend(
+                    self._scan(
+                        self._dast_templates[keyword],
+                        ScanUsing.TEMPLATES,
+                        dast_targets[keyword],
+                        extra_nuclei_args=["-dast"],
+                    )
+                )
+
+            dast_targets["other"] = []
+            for item in link_package:
+                if item:
+                    param_url = item
+                    for keyword, template_data in DAST_SCANNING.items():
+                        param_url = add_common_params_from_wordlist(
+                            item, template_data["params_wordlist"], template_data["param_default_value"]
```

Comment:
this will set param_url to the item with parameters added for the last dict entry, right?